src/ResourceManager/Network/Commands.Network/NetworkProfile/IpConfigurationProfile/NewAzureNetworkProfileContainerNetworkInterfaceConfigIpConfigCommand.cs

```diff
@@ -0,0 +1,61 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.Network.Models;
+using Microsoft.Azure.Management.Network.Models;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Management.Automation;
+
+namespace Microsoft.Azure.Commands.Network
+{
+    [Cmdlet(VerbsCommon.New, ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "NetworkProfileContainerNicConfigIpConfig", SupportsShouldProcess = true), OutputType(typeof(PSContainerNetworkInterfaceConfiguration))]
+    public partial class NewAzureNetworkProfileContainerNetworkInterfaceConfigIpConfigCommand : AzureNetworkProfileContainerNetworkInterfaceConfigIpConfigBase
```

Comment:
where are Add-, Set-, REmove and Get- for this?

The [changes](https://github.com/Azure/azure-powershell/pull/7225/files#diff-b60b2559dbc8dde377acf5fe189d4668R373) suggestthat the following were added :
``` 
               'Get-AzureRmNetworkProfileContainerNicConfigIpConfig', 
               'New-AzureRmNetworkProfileContainerNicConfigIpConfig', 
               'Add-AzureRmNetworkProfileContainerNicConfigIpConfig', 
               'Remove-AzureRmNetworkProfileContainerNicConfigIpConfig', 
               'Set-AzureRmNetworkProfileContainerNicConfigIpConfig', 
```

but I only see New